src/test/tests/databases/export_db.py

```diff
@@ -126,15 +134,107 @@ def VTK_check_binary(filename):
     b = "BINARY" == line[0:6]
     return b
 
-def cleanup_files():
-    exts = ("*.ply", "*.raw", "*.stl", "*.silo", "*.tec", "*.okc", "*.visit", "*.xyz", "*.obj")
-    files = os.listdir(".")
-    for f in files:
-        for e in exts:
-            pattern = e[1:]
-            if f[-len(pattern):] == pattern:
-                os.unlink(f)
-                break
+
+# Test exporting some surfaces in other formats and plotting the results.
+# e = ExportDBAttributes
+# v = View3D
+# a = AnnotationAttributes
+# count = current test number (used for test names)
+# testNum = 1 or 2 (for test1 or test2 which tests write groups)
+
+def export_and_plot(e, v, a, count, testNum):
+
+    # testNum ==1 no write groups
```

Comment:
Would it be onerous to change `int testNum` to `bool useWriteGroups`? Maybe it could make the coding a little clearer?

Reply:
Done.